docs/user_docs.ipynb

```diff
@@ -159,8 +162,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "observation = scarlet.Observation(images, psfs)\n",
-    "scene = scarlet.Scene(images.shape, psfs=target_psf)"
+    "frame = scarlet.Frame(images.shape, psfs=model_psf, bands=filters)\n",
+    "observation = scarlet.Observation(images, psfs=psfs, bands=filters).match(frame)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. note::\n",
```

Comment:
You cannot insert an RST `note` or `warning` in a cell, because ipython uses markdown, not restructured text. This is one of the PITA parts of writing documents using ipython. So you have to put the note in its own cell, choose cell type = raw, view the type of raw input cell, and select rst from the dropdown, all in ipython, otherwise the note isn't rendered properly (see the built docs).

Reply:
Moved to ordinary markdown